youtube2zim/scraper.py

```diff
@@ -699,7 +751,7 @@ def update_metadata(self):
             width=48,
             height=48,
             method="thumbnail",
-            to=self.build_dir.joinpath("favicon.jpg"),
+            dst=self.build_dir.joinpath("favicon.jpg"),
```

Comment:
hum, so that was broken since moving to new image module I guess

Reply:
Yup